interface/new/new_comprehensive.php

```diff
@@ -22,8 +26,60 @@
 $searchcolor = empty($GLOBALS['layout_search_color']) ?
   '#ffff55' : $GLOBALS['layout_search_color'];
 
-$WITH_SEARCH = ($GLOBALS['full_new_patient_form'] == '1' || $GLOBALS['full_new_patient_form'] == '2');
-$SHORT_FORM  = ($GLOBALS['full_new_patient_form'] == '2' || $GLOBALS['full_new_patient_form'] == '3');
+$WITH_SEARCH = ($GLOBALS['full_new_patient_form'] == '1' || $GLOBALS['full_new_patient_form'] == '2' || $GLOBALS['full_new_patient_form'] == '4');
+$SHORT_FORM  = ($GLOBALS['full_new_patient_form'] == '2' || $GLOBALS['full_new_patient_form'] == '3' );
+
+//this will hold the javascript submit me param new_validate;
+$new_validate = $GLOBALS['new_validate'] ? 1 : 0;
+if($GLOBALS['full_new_patient_form'] == '4')//use hook of patient validation = 4
+{
+    $hook = checkIfPatientValidationHookIsActive();
+
+    if($hook){
+
+        print "<script>";
+        //ajax call to the zend PatientValidation controller.
+        print "window.onload = function () {
+        
+                       
+            
+           document.getElementById('create').addEventListener(\"click\", function(e) {
+            e.stopPropagation();
+           var submitmeValidation =  submitme(".$new_validate.",e,\"DEM\") ;
+                 if(submitmeValidation === true)
+                 {
+                $.ajax({
+                        url: '".$GLOBALS['web_root']."/interface/modules/zend_modules/public/patientvalidation',
```

Comment:
Just out of curiosity. What benefit does using Zend bring you?(I am guessing there are things you are using within Zend for the validation stuff)